mono/profiler/log.c

```diff
@@ -1303,8 +1303,10 @@ process_heapshot (void)
 #define ALL_GC_EVENTS_MASK (PROFLOG_GC_EVENTS | PROFLOG_GC_MOVE_EVENTS | PROFLOG_GC_ROOT_EVENTS)
 
 static void
-gc_event (MonoProfiler *profiler, MonoProfilerGCEvent ev, uint32_t generation)
+gc_event2 (MonoProfiler *profiler, MonoProfilerGCEvent ev, uint32_t generation, gboolean is_serial)
```

Comment:
We don't really *have* to rename this function, but if you do, there are some references to it in comments throughout this file that need to be updated.